team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -35,6 +35,9 @@
 public class MonteCarloPlayer extends LocalPlayer {
   private final float MAX_RUNTIME = 10;
 
+  //The constant used for UCB function. Same one chosen in the John Levine video.
+  private final double EXPLORATIONS = Math.sqrt(2);
```

Comment:
This name is confusing - this is the constant used to balance between the value of a node and the number of visits ("exploitation vs exploration" according to John Levine). Maybe rename this to `EXPLORATION_FACTOR` or something similar, to indicate its meaning better.